index.html

```diff
@@ -0,0 +1,140 @@
+                <li><a href="#" class="links">Join</a></li>
+                <li><a href="#sponsors" class="links">Sponsor</a></li>
+                <li><a href="#" class="links">News</a></li>
+                <li><button type="button">JSI Conference</button></li>
+            </ul>
+        </div>
+    </nav>
+  </header>
+  <main>
+      <!--  INTRODUCTION SECTION  -->
+      <section class="intro">
+          <div>
+             <h2>"Hello! Sharing world"</h2>
+             <h1>International Javascript Conference 2022</h1>
+          </div>
+          <article>
+              <p class="intro-txt">International javascript Conference is a summit that talks about the latest innovations in javascript.This is to help broaden the knowledge of web developers and javascript enthusiast</p>
+          </article>
+          <div>
+              <h3>From April 25th ~ 28th, 2022</h3>
+              <p>@ The O2 Arena. Peninsula Square, London SE10 0DX, United Kingdom</p>
+          </div>
+      </section>
+      <!--  MAIN PROGRAM SECTION  -->
+      <section class="program" id="program">
+          <div>
+         <h3 class="p-title">Main Program</h3>
+          </div>
+          <article class="program-card">
+              <div class="card">
+                 <i class="fas fa-book-reader"></i>
+                 <h3>Lecture</h3>
+                 <p>Listen to speakers from various countries about the latest innovations in javascript.</p>
+              </div>
+              <div class="card">
+                 <i class="fas fa-cogs"></i>
+                 <h3>Language Exhibition</h3>
+                 <p>Witness experts give notes and also perform live coding.</p>
+              </div>
+              <div class="card">
+                 <i class="far fa-comments"></i>
+                 <h3>Forum</h3>
+                 <p>Join a community of like-minded individuals and share ideas</p>
+              </div>
+              <div class="card">
+                 <i class="fas fa-laptop-code"></i>
+                 <h3>Workshop</h3>
+                 <p>Our workshop will cover a hands-on session on javascript and its frameworks</p>
+              </div>
+              <div class="card">
+                 <i class="far fa-handshake"></i>
+                 <h3>Networking</h3>
+                 <p>Get an opportunity to network with javascript experts around the globe.</p>
+              </div>
+          </article>
+          <div class="program-btn">
+              <button type="button">Join IJS Conference</button>
+             <a href="index.html">SEE THE WHOLE PROGRAM</a>
+         </div>
+      </section>
+      <!--  FEATURED SPEAKERS SECTION  -->
+      <section class="featured-speakers">
+         <div>
+         <h3 class="f-title">Featured Speakers</h3>
+         </div>
+         <article class="speakers">
+         <!--  SPEAKER CARDS ARE CREATED IN JAVASCRIPT  -->
+         </article>
+      </section>
+      <!--  SPONSORS SECTION  -->
```

Comment:
- [ ] Finally, in the `sponsors` section the items are should be more centered. Let's compare again 👇 

- Your design 👇 
![footer-st](https://user-images.githubusercontent.com/67087664/147965996-824f58c8-04d6-49ec-8d34-075243458d4c.png)

- Original 👇 
![footer](https://user-images.githubusercontent.com/67087664/147966058-8a364a24-4dcb-4cd8-8f8d-0367f11067b4.png)

Kindly, fix it too 😅